frontend/src/Pages/MeetTheTeamPage/style.css

```diff
@@ -0,0 +1,169 @@
+.centeredText {
+    text-align: center;
+}
+
+.MeetTheTeamPage {
+    width: 1100px;
+    margin: auto;
+}
+
+#meet-our-team-header {
+    font-family: Mokoto;
+    font-size: 46px;
+    margin-bottom: 40px;
+}
+
+.director-jrDirector-header {
+    font-family: Mokoto;
+    font-size: 30px;
+    margin-bottom: 33px;
+}
+
+.director-row {
+    width: 100%;
+    justify-content: center;
+    margin-bottom: 15px;
+    /*column-gap: 30px;*/
+}
+
+
+.director-div {
+    margin-bottom: 5px;
+    display: grid;
+    row-gap: 13px;
+}
+
+.director-img {
+    width: 200px;
+    height: 200px;
+    object-fit: cover;
+    border-radius: 50%;
+    border: 4px solid white;
+}
+
+.director-avatar {
+    width: 70px;
+    height: 70px;
+    object-fit: cover;
+    border-radius: 50%;
+    border: 2px solid #910307;
+    background-color: #353535;
+    display: flex;
+    position: absolute;
+    justify-self: flex-end;
+    align-self: flex-end;
+}
+
+.director-name-div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+}
+
+.name {
+    color: white;
+    font-size: 1.8rem;
+    white-space: nowrap;
+}
+
+.position {
+    color: #910307;
```

Comment:
This color is a little hard to read on the dark background is there another color we can use to make it more visible?